src/app/global-vars.service.ts

```diff
@@ -717,28 +730,63 @@ export class GlobalVarsService {
     this.amplitude.logEvent(event, data);
   }
 
-  launchLoginFlow() {
-    this.logEvent("account : login : launch");
-    this.identityService.launch("/log-in").subscribe((res) => {
-      this.logEvent("account : login : success");
-      this.backendApi.setIdentityServiceUsers(res.users, res.publicKeyAdded);
-      this.updateEverything().subscribe(() => {
-        this.flowRedirect(res.signedUp);
-      });
+  openJumio(jumioSuccessRoute: string, jumioErrorRoute: string): void {
+    // Note: this endpoint will fail if success and error routes do not conform to the expectations of Jumio.
+    // See here for details: https://github.com/Jumio/implementation-guides/blob/master/netverify/netverify-web-v4.md#url-requirements
+    this.backendApi
+      .JumioBegin(
+        environment.jumioEndpointHostname,
+        this.loggedInUser?.PublicKeyBase58Check,
+        jumioSuccessRoute,
+        jumioErrorRoute
+      )
+      .subscribe(
+        (res) => {
+          window.open(res.URL);
+        },
+        (err) => {
+          this._alertError(err);
+        }
+      );
+  }
+
+  // Helper to launch the get free clout flow in identity.
+  launchGetFreeCLOUTFlow() {
+    this.logEvent("identity : jumio : launch");
+    this.identityService.launch("/get-free-clout").subscribe(() => {
+      this.logEvent("identity : jumio : success");
+      this.updateEverything();
     });
   }
 
-  launchSignupFlow() {
-    this.logEvent("account : create : launch");
+  launchIdentityFlow(event: string): void {
+    this.logEvent(`account : ${event} : launch`);
     this.identityService.launch("/log-in").subscribe((res) => {
-      this.logEvent("account : create : success");
+      this.logEvent(`account : ${event} : success`);
       this.backendApi.setIdentityServiceUsers(res.users, res.publicKeyAdded);
-      this.updateEverything().subscribe(() => {
-        this.flowRedirect(res.signedUp);
+      let updateFlowFinishedObservable = res.jumioSuccess
+        ? this.backendApi.JumioFlowFinished(
```

Comment:
Shouldn't this call happen from Identity?

Reply:
moving now.